test/e2e/additional_routes_test.go

```
@@ -0,0 +1,377 @@
+
+		checkOAuthRedirectURI(t, client, hostnameA, false)
+		checkOAuthRedirectURI(t, client, hostnameB, true)
+		checkConfigMapAdditionalHost(t, client, hostnameA, false)
+		checkConfigMapAdditionalHost(t, client, hostnameB, true)
+	})
+
+	t.Run("RouteAdmission", func(t *testing.T) {
+		err := wait.Poll(2*time.Second, additionalRoutePollTimeout, func() (bool, error) {
+			route, err := client.Routes.Routes(api.OpenShiftConsoleNamespace).Get(context.TODO(), nameB, metav1.GetOptions{})
+			if err != nil {
+				return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 72) but it returns nil

(nilerr)